llvm/include/llvm/MC/MCDirectives.h

```diff
@@ -50,14 +50,6 @@ enum MCSymbolAttr {
   MCSA_Memtag,                  ///< .memtag (ELF)
 };
 
-enum MCAssemblerFlag {
```

Comment:
Nice! I agree that it's better to handle these very target-specific flags into XXXTargetStreamer